.github/workflows/push.yml

```diff
@@ -96,16 +96,23 @@ jobs:
         run: |
           # Exit with status code 1 if there are differences (i.e. unformatted files)
           git diff --exit-code
-      - name: golangci-lint
+      - name: Run lint checks
```

Comment:
Do we need to run it using a github action if we are running it from `make fmtcheck` as well?

Reply:
`make fmt` is running `golangci-lint fmt` which is different from `golangci-lint run` . in v2 they splitted linting and formatting functionality into two different subcommands